packages/config/.eslintrc

```diff
@@ -2,5 +2,8 @@
     "env": {
         "browser": false,
         "node": true
+    },
+    "rules": {
+        "@typescript-eslint/no-var-requires": "off"
```

Comment:
Are we sure we want to turn this off globally for the entire monorepo? Or should we disable it per-case basis?

Reply:
It's only for the `config` package, not the monorepo.